### PR TITLE
feature: [ST-3320] send response status code to html swapper

### PR DESCRIPTION
**Pull request by bstewart00**
https://wovnio.atlassian.net/browse/ST-3320
https://github.com/WOVNio/html-swapper/pull/1044

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@
     <groupId>com.github.wovnio</groupId>
     <artifactId>wovnjava</artifactId>
     <name>wovnjava</name>
-    <version>2.0.0</version>
+    <version>2.0.1</version>
```
Comment (bryanzerrudo) [inline pom.xml:10]: Perhaps could use minor version?
```suggestion
    <version>2.1.0</version>
```

I think "sending a new parameter to an API" can be considered more as a feature than a bugfix.

Comment (bstewart00) [reply pom.xml:10]: it's not really a public API from wovnjava's perspective

Comment (bryanzerrudo) [reply pom.xml:10]: Wordpress ([PR](https://github.com/WOVNio/wordpress_plugin/pull/124)) uses minor version though. I think we should change one or the other.

Comment (bstewart00) [reply pom.xml:10]: I don't think that was strictly correct, but for consistency i'll just use minor then

**Review by bryanzerrudo — APPROVED** [review]

**Review by Soichiro725 — APPROVED** [review]

**Pull request merged**